pkg/Directory.Build.props

```diff
@@ -12,207 +11,38 @@
-  </PropertyGroup>
-
-  <!-- Most packages need the host to be built first. -->
-  <ItemGroup Condition="'$(DisableOrderDependencies)' != 'true'">
-    <OrderProjectReference Include="@(CorehostProjectToBuild)" />
-    <!-- Pkgproj requires its depproj crossgen outputs to be signed.-->
-    <OrderProjectReference
-      Condition="'$(MSBuildProjectExtension)' == '.pkgproj'"
-      Include="$(RepoRoot)signing\SignBinaries.proj" />
-  </ItemGroup>
-
-  <!-- In *.builds projects, the current phase's name is the same as the project name. -->
-  <PropertyGroup>
-    <BuildPhase>$(MSBuildProjectName)</BuildPhase>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(PackageTargetRuntime)' == ''">
-    <SkipValidatePackage>true</SkipValidatePackage>
-    <IncludeRuntimeJson Condition="'$(IncludeRuntimeJson)' == ''">true</IncludeRuntimeJson>
-    <IsLineupPackage Condition="'$(IsLineupPackage)' == ''">true</IsLineupPackage>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(MSBuildProjectExtension)' == '.depproj'">
-    <!--
-      we don't want any analyzers by ResolveNuGetPackageAssets
-      null-refs when this isn't set and an analyzer is in the packages
-    -->
-    <Language>unused</Language>
-    <NuGetTargetMoniker>$(NETCoreAppFrameworkMoniker)</NuGetTargetMoniker>
-    <NuGetTargetMonikerShort>$(NETCoreAppFramework)</NuGetTargetMonikerShort>
-    <PackageTargetFramework>$(NETCoreAppFramework)</PackageTargetFramework>
-    <CrossGenOutputPath>$(CrossGenRootPath)$(MSBuildProjectName)\</CrossGenOutputPath>
-    <ContainsPackageReferences>true</ContainsPackageReferences>
-    <RidSpecificAssets Condition="'$(RuntimeIdentifier)' != ''">true</RidSpecificAssets>
-    <CrossGenSymbolsOutputPath>$(PackageSymbolsBinDir)/$(MSBuildProjectName)</CrossGenSymbolsOutputPath>
   </PropertyGroup>
-
-  <PropertyGroup>
-    <TargetFramework Condition="'$(TargetFramework)' == ''">$(PackageTargetFramework)</TargetFramework>
-    <TargetFramework Condition="'$(TargetFramework)' == ''">$(NETCoreAppFramework)</TargetFramework>
-  </PropertyGroup>
-
-  <!-- Add required legal files to packages -->
-  <ItemGroup Condition="'$(MSBuildProjectExtension)' == '.pkgproj'">
-    <File Condition="Exists('$(PackageLicenseFile)')"
-          Include="$(PackageLicenseFile)" >
-      <SkipPackageFileCheck>true</SkipPackageFileCheck>
-    </File>
-    <File Condition="Exists('$(PackageThirdPartyNoticesFile)')"
-          Include="$(PackageThirdPartyNoticesFile)" >
-      <SkipPackageFileCheck>true</SkipPackageFileCheck>
-    </File>
-  </ItemGroup>
-
-  <PropertyGroup>
-    <RIDPropsFile Condition="'$(RIDPropsFile)' == ''">$(MSBuildThisFileDirectory)\windowsdesktopRIDs.props</RIDPropsFile>
-  </PropertyGroup>
-
-  <Import Project="$(RIDPropsFile)" />
-
+  
+  <!--
+    SDK currently does not know about net6.0 TargetFramework.
+    We need to manually add these to correctly resolve the targeting packs.
+    https://github.com/dotnet/sdk/issues/13950  
```

Comment:
This appears to have been resolved.

Reply:
To remove these items, we need to update the global.json to reference a 6.0-based SDK.